lib/membrane_ffmpeg_video_filter/text_overlay.ex

```diff
@@ -8,13 +8,25 @@ defmodule Membrane.FFmpeg.VideoFilter.TextOverlay do
   Additionally, the element has to receive proper caps with picture format and dimensions.
   """
   use Membrane.Filter
+
+  require Membrane.Logger
+
   alias __MODULE__.Native
-  alias Membrane.Buffer
   alias Membrane.Caps.Video.Raw
 
   def_options text: [
                 type: :binary,
-                description: "Text to be displayed on video"
+                description:
+                  "Text to be displayed on video. Either text or text_stream must be provided",
+                default: nil
+              ],
+              text_stream: [
```

Comment:
I'd rename that to `text_intervals` unless you have a better idea - I think `text_stream` focuses too much on the form of passed value rather then on meaning